de.metas.adempiere.adempiere/client/src/main/java/de/metas/process/ui/ProcessPanel.java

```diff
@@ -185,19 +187,38 @@ public Dimension getPreferredSize()
 
 	private final CButton bOK = ConfirmPanel.createOKButton(true);
 	private final CButton bBack = ConfirmPanel.createBackButton(true);
-	
-	/*package*/void installTo(final ProcessPanelWindow window)
+
+	/**
+	 * #782: Focus on the first parameter of the process when the process panel is opened, so it can be directly changed from keyboard
+	 */
+	private final WindowListener onWindowOpenFocusFirstParameter = new WindowAdapter()
+	{
+		@Override
+		public void windowOpened(final WindowEvent e)
+		{
+			if(parameterPanel == null)
```

Comment:
also, i would suggest to request focus only if CARDNAME_ProcessParameters.equals(getCurrentCardName()).

I would also extract the code from here and have ProcessPanel.requestFocusInWindow() method. Here, just call it.